org-noter-core.el

```diff
@@ -1845,6 +1858,19 @@ See `org-noter-notes-window-behavior' for more information."
               (org-entry-put nil org-noter--property-behavior (format "%s" chosen-behaviors))
             (org-entry-delete nil org-noter--property-behavior))))))))
 
+(defun org-noter-toggle-notes-window-location ()
```

Comment:
I feel like I'm nitpicking so feel free to ignore -- is it "location" or "orientation"?